source/public/index.html

```diff
@@ -5,11 +5,86 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="/styles/styles.css">
-    <title>Document</title>
+    <title>Digital Dr3am T3am - Journal Homepage</title>
 </head>
 
 <body>
+    <header>
+        <h1>Welcome back to your developer journal!</h1>
+    </header>
 
+    <main>
+        <!-- Calendar -->
+        <section id="calendar">
+            <div class="calendar">
+                <div class="header">
+                    <button type="button" title="Previous month">
+                        <i class="fa-solid fa-chevron-left"></i>
+                    </button>
+                    <div class="month-year" id="month-year"></div>
+                    <button type="button" title="Next month">
+                        <i class="fa-solid fa-chevron-right"></i>
+                    </button>
+                </div>
+                <div class="days">
+                    <div class="day">Mon</div>
+                    <div class="day">Tue</div>
+                    <div class="day">Wed</div>
+                    <div class="day">Thu</div>
+                    <div class="day">Fri</div>
+                    <div class="day">Sat</div>
+                    <div class="day">Sun</div>
+                </div>
+                <div class="dates" id="dates"></div>
+            </div>
+        </section>
+
+        <!-- Project modal -->
```

Comment:
Could you move the modal to the bottom of the HTML (below the section to display the projects) for consistency with the journal page?